src/Query/Transformer.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Component\Database\Query;
+
+interface Transformer
+{
```

Comment:
Missing docblock for `@throws InvalidQueryException`.

Reply:
Do you mean the interface should be aware of what implementations do? Or should we add such doc block to implementations? e.g. `Transformer\NamedArrayParameter::execute()`?

Reply:
Thinking that way is better we put it in `Transformer\NamedArrayParameter::execute()` since not all of the classes follow the interface would throw the exception.

Reply:
Any transformer would likely throw the exception. It should go here and not only in the specific transformer. You have to do this because you have to document what Transformers are potentially going to throw, so the caller can handle it.

Reply:
done.